pkg/operator/discover/discover.go

```diff
@@ -212,10 +274,46 @@ func ListDevices(context *clusterd.Context, namespace, nodeName string) (map[str
 			logger.Warningf("failed to unmarshal %s", deviceJson)
 			continue
 		}
-		devices[node] = d
+		for i := range d {
+			devices = append(devices, d[i])
+		}
 	}
-	logger.Debugf("devices %+v", devices)
+	logger.Debugf("devices in use %+v", devices)
```

Comment:
It'd be nice to know by which osd maybe?

Reply:
this is for debugging. For production diagnose, we can get that info from configmap.